integrationTests/multiShard/block/executingMiniblocksSc_test.go

```diff
@@ -308,7 +313,9 @@ func TestShouldProcessWithScTxsJoinNoCommitShouldProcessedByValidators(t *testin
 	syncBlock(t, nodes, idxProposers, round)
 	round = incrementAndPrintRound(round)
 
-	nodeJoinsGame(nodes, idxProposerShard0, topUpValue, hardCodedScResultingAddress)
+	nonceNodeIdx0 := uint64(0)
```

Comment:
this has to be refactored together with the whole file

Reply:
Yes, @BeniaminDrasovean is doing this. I just patch the file in order to get the task done.